controllers/articles/getArticles.controllers.js

```diff
@@ -1,4 +1,6 @@
-const { getArticlesModel } = require("../models/getArticles.models");
+const {
+  getArticlesModel,
+} = require("../../models/articles/getArticles.models");
 
 exports.getArticles = (req, res, next) => {
```

Comment:
This looks like a getSingleArticle function... I'd call it getArticle or similar.